soc/riscv/riscv-privilege/telink_b91/linker.ld

```diff
@@ -17,16 +17,6 @@ MEMORY
 	RAM_ILM  (rwx) : ORIGIN = DT_REG_ADDR(DT_NODELABEL(ram_ilm)), LENGTH = DT_REG_SIZE(DT_NODELABEL(ram_ilm))
 }
 
-SECTIONS
-{
-	SECTION_PROLOGUE(vector,,)
-	{
-		. = CONFIG_ROM_START_OFFSET;
-		. = ALIGN(4);
-		KEEP(*(.init.*))
-	} GROUP_LINK_IN(ROM_INIT)
```

Comment:
should remove `ROM_INIT (rx)  : ORIGIN = DT_REG_ADDR(DT_CHOSEN(zephyr_flash)), LENGTH = DT_REG_SIZE(DT_CHOSEN(zephyr_flash))` in L16 as well

Reply:
Thanks! I removed it.